wiki/Article_Style_Guide/en.md

```diff
@@ -101,20 +138,29 @@ or
 You can choose either the underline style or hashtag style of level 1 and level 2 headings.
 Note that heading levels 3 and onwards use the hashtag style.
 
-### Article Titles
-
-Titles are to be in title case.
-The title of an article is the name of the folder that the article is located in.
+Optionally, but preferably, add an extra line before and after the headings.
+Doing so will help make the headings stand out a bit more when editing.
 
-For English article titles, if you need to reword the title, you must rename the folder to match the article title.
+### Article Titles
 
-All article titles are to be using the level 1 heading.
+All articles must have **one** level 1 heading, being the article's title.
 For example:
 
 ```
 # Folder Name
```

Comment:
iirc, this was to say, use the folder name as the article title (for English and untranslatable titled articles), otherwise there will be two names that we have to remember...? ¯\\\_(ツ)\_\/¯